src/test/java/nl/tudelft/contextproject/controller/GameControllerTest.java

```diff
@@ -57,26 +60,34 @@ public void setUp() {
 		controller.setRootNode(rootNode);
 		inputManager = mock(InputManager.class);
 		controller.setInputManager(inputManager);
-		
+
 		LinkedList<Entity> entities = new LinkedList<>();
 		Entity entity = mock(Entity.class);
 		when(entity.getSpatial()).thenReturn(mock(Spatial.class));
 		when(entity.getState()).thenReturn(EntityState.ALIVE);
+		
+		PhysicsControl geom = mock(PhysicsControl.class);
+		phe = mock(BulletAppState.class);
+		PhysicsSpace phs = mock(PhysicsSpace.class);
+		when(entity.getPhysicsObject()).thenReturn(geom);
+		when(phe.getPhysicsSpace()).thenReturn(phs);
+		
 		entities.add(entity);
 		VRPlayer player = mock(VRPlayer.class);
 		when(player.getSpatial()).thenReturn(mock(Spatial.class));
-		
+
 		MazeTile t = mock(MazeTile.class);
 		when(t.getSpatial()).thenReturn(mock(Spatial.class));
 		MazeTile[][] tiles = {{t, t}, {null, t}, {t, null}};
 		LinkedList<Light> lights = new LinkedList<>();
 		lights.add(light);
 		level = new Level(tiles, lights);
 		game = new Game(level, player, entities);
-		
+
 		controller.setGame(game);
+		controller.setPhysicsEnvironmentNode(phe);
 	}
```

Comment:
Ouch, this setup method is getting pretty extensive. This could possibly be cleaned up when we have more time.